tests/Form/SelectTest.php

```diff
@@ -13,6 +13,12 @@ public function testMultiselect()
         $page = $this->getSession()->getPage();
         $this->assertEquals('Multiselect Test', $webAssert->elementExists('css', 'h1')->getText());
 
+        $selectWithoutOption = $webAssert->fieldExists('select_without_option');
+        $this->assertNull($selectWithoutOption->getValue());
```

Comment:
In a browser, running `document.querySelector('[name="select_without_option"]').value` gives me an empty string as value in that case, which make this test fail in the Selenium2Driver

Reply:
Damn you're right. Want me to submit a PR ?

Reply:
please do it if possible. Otherwise I'll do the change myself.

Reply:
See #45